### ISSUE TITLE
Pyright ignoring PyQt5 type errors that are recognized by mypy

### ISSUE THREAD

**Issue by glutanimate** [microsoft/pyright#348]
**Describe the bug**

Pyright does not seem to recognize PyQt5 typing errors which are recognized by mypy.

**To Reproduce**

1. Install mypy, PyQt5, and PyQt5-stubs with:

         pip install PyQt5-stubs mypy

2. Save the code below as `test.py`:

    ```python
    from PyQt5.QtWidgets import QDialog

    a = 2

    def update_dialog(dialog: QDialog) -> QDialog:
        return dialog

    update_dialog(a)
    ```

3. Run mypy and pyright over the code in turn:

        mypy test.py
        pyright test.py

**Expected behavior**

`mypy` correctly reports:

    test.py:7: error: Argument 1 to "update_dialog" has incompatible type "int"; expected "QDialog"
    Found 1 error in 1 file (checked 1 source file)

`pyright` should also recognize that issue, but actually returns with:

    Searching for source files
    Found 1 source file
    0 errors, 0 warnings 
    Completed in 1.188sec

**VS Code extension or command-line**

Tested both with the CLI executable and VS Code extension. Version is 1.1.2 in both cases.

**Additional context**

*OS*

    $ lsb_release -rd
    Description:    Ubuntu 18.04.3 LTS
    Release:        18.04

    $ uname -a
    Linux UBN 4.15.0-70-generic #79-Ubuntu SMP Tue Nov 12 10:36:11 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

*Python*

    $ python --version
    Python 3.6.7

*Pip*
    
    $ pip freeze
    mypy==0.740
    mypy-extensions==0.4.3
    PyQt5==5.13.1
    PyQt5-sip==12.7.0
    PyQt5-stubs==5.13.1.3
    typed-ast==1.4.0
    typing-extensions==3.7.4.1

That's all the pertinent info I can think of, but please let me know if there is anything else I can add to help with troubleshooting. 

Thanks a lot for your work pyright so far, and thanks in advance for giving this a look!

**Comment by erictraut** [comment]
Thanks for the bug report. PyQt5 is using a provision of PEP 561 to ship its type stubs separately from the main package within a parallel directory structure that shares the package name but has "-stubs" appended. Pyright didn't support this provision previously (because I wasn't aware of it until now). I've added support for it. This will be fixed in the next version of pyright.

**Comment by erictraut** [comment]
This is now implemented in 1.1.3, which I just published.

**Comment by glutanimate** [comment]
Fantastic! Thanks for the lightning-fast fix!
